jekyll/_cci2/build-project.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: classic-docs
+title: "Build a Project"
```

Comment:
Consider 'Add a Project' or 'Setup Your Project' -- trying to get away from the word "build" when thinking about a whole project - "build" is one job among several, especially once workflows are a thing.